aten/src/ATen/native/native_functions.yaml

```diff
@@ -14580,3 +14580,6 @@
   dispatch:
     CUDA: _fused_adamw_kernel_cuda_
   autogen: _fused_adamw, _fused_adamw.out
+
+- func: _propagate_xla_data(Tensor input, Tensor output) -> ()
```

Comment:
@bdhirsh I have added the op you suggested in https://github.com/pytorch/xla/pull/4505#issuecomment-1405176038.  Please review. You can just check the commit called: [Functionalization] Adds _propagate_xla_data.